libafl_qemu/src/modules/usermode/redirect_stdin.rs

```diff
@@ -77,7 +81,7 @@ where
     ) where
         ET: EmulatorModuleTuple<I, S>,
     {
-        assert!(self.input_addr.is_some());
+        assert!(!self.input_addr.is_null());
```

Comment:
I still don't understand why this would be the case, since you set the input_addr in the harness(?)
The assert should be the other way around if anything

Reply:
deleted

Reply:
if it is null the syscall will just continue as if nothing happened so it's fine to remove it